### ISSUE TITLE
[Feature] Chat Log on Client side

### ISSUE THREAD

**Issue by hjanuschka** [juliancwirko/s-chat-app#8]
is it possible to enable chat-log/history on client side, based on a cookie?

e.g: user chats to me, writes a issue, he then reloads the page, i reply via admin interface, he never gets the reply.

nice project, if it is not possible right, now i may do a pr

regards

**Comment by juliancwirko** [comment]
Hi, yes it needs something like that. For now it is impossible to use it on websites which aren't single page apps like those angular, react based apps. I think we need to store user's uniq id (called `userSessionId`) in the browser's localstorage and then reconnect the session on reload if it exists in localstorage instead of creating the new one. This could be some more work, but of course if you want you can send PR. We would need to do this for Blaze package and also static js logic. Thanks.

**Comment by juliancwirko** [comment]
From version 1.2.0 the chat history preserves after website refresh or changing routes, it will create new session only when you close and reopen your website

**Comment by juliancwirko** [comment]
I'll close it for now. It is implemented. If you want you can still comment in this topic. Thanks!

**Comment by anedisi** [comment]
what about using localstorage instead of sessionstorage.

if i open a new tab with sessionstorage im creating a new session id and with a localstorage im using the old one. 

in ui it would look like im starting a new chat with sessionstorage and resuming a chat with localstorage. some people open multiple tabs

now the problem is when to clear the sessionid from localstorage.

**Comment by juliancwirko** [comment]
Yes, you're right it might be problematic. I need to think about it.
